tests/integration/component.go

```diff
@@ -88,11 +88,6 @@ func componentTests(args ...string) {
 			applicationName := helper.GetConfigValue("Application")
 			Expect(applicationName).To(Equal("app"))
 
-			// clean up
-			helper.CmdShouldPass("odo", "app", "delete", "app", "-f")
-			helper.CmdShouldFail("odo", "app", "delete", "app", "-f")
-			helper.CmdShouldFail("odo", "component", "delete", componentName, "-f")
```

Comment:
This is also a verification steps of component delete should fail if the application itself is not present